.vscode/settings.json

```diff
@@ -9,4 +9,4 @@
   },
   "slither.solcPath": "",
   "slither.hiddenDetectors": []
-}
\ No newline at end of file
+}
```

Comment:
We should define formatting rules in (`package.json?`) so that we all use the same format when running the `format` command. 
Otherwise, we’ll keep committing files that differ only in formatting, which makes PRs harder to review.